### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by ibr5500**
### In this [PR] we (@GOlivierNation, @ibr5500) have:

> - Create DataBase based on the following diagram.
clinic_diagram

> - Create the following tables (Patients, medical_histories, treatments, invoices, and invoices_items).
> - Update the README file.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  total_price NUMERIC, 
+  invoice_id INT, 
+  CONSTRAINT fk_invoice
+  FOREIGN KEY (invoice_id)
+  REFERENCEs invoices(id),
+    treatment_id INT, 
+	CONSTRAINT fk_treatment
+	FOREIGN KEY (treatment_id)
+	REFERENCES treatments(id)
+);
```
Comment (ShadyShawkat) [inline schema_based_on_diagram.sql:57]: - [x] Kindly implement the relationship table between `medical_histories` and `treatments` tables and apply their foreign keys. (As there is a many-to-many relationship between the `medical_histories` and `treatments` tables)
- [x] Please make sure to implement indices for all of the foreign keys.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (ibr5500) [reply schema_based_on_diagram.sql:57]: Resolved

**Comment by ibr5500** [comment]
Thank you @ShadyShawkat for the code review.

**Review by Kingstalux — APPROVED** [review]
Hi @team,
# Status: Approved ✔️ ✔️ ✔️ 
 
download


Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Thank you @Kingstalux for the code review.

**Pull request merged**